src/App.js

```diff
@@ -1,19 +1,16 @@
 import React from "react";
```

Comment:
This is why this import can't be removed: https://medium.com/@Zwenza/why-do-you-need-to-import-react-in-functional-components-7385e4329ffb